tests/x11/rmt/rmt_chinese.pm

```diff
@@ -0,0 +1,81 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: yast2-rmt rmt-server
+# Summary: rmt-cli internationalization / localization test (only test Chinese)
+# Steps: 1. Set language to simply Chinese.
+#        2. Install and configure RMT server, ensure the configuration process
+#           in Chinese.
+#        3. Run rmt-cli sync and ensure the output is in Chinese.
+#
+# Maintainer: Lemon Li <leli@suse.com>
+
+use strict;
+use warnings;
+use testapi;
+use base 'x11test';
+use repo_tools;
+use utils;
+use x11utils;
+use version_utils;
+
+sub set_language_to_Chinese {
+    if (check_var('DESKTOP', 'gnome')) {
+        select_console 'root-console';
+        turn_off_gnome_screensaver_for_gdm;
+    }
+    select_console 'x11', await_console => 0;
+    wait_still_screen 15;
+    ensure_unlocked_desktop;
+    assert_screen "generic-desktop";
+
+    x11_start_program('xterm');
+    wait_still_screen 2, 2;
+    become_root;
+    script_run('yast2 language');
+    send_key_until_needlematch 'yast2-lang-simplified-chinese', 'down', 180;
+    send_key 'alt-o';
+
+    # Problem here is that sometimes installation takes longer than 10 minutes
+    # And then screen saver is activated, so add this step to wake
+    my $timeout = 0;
+    until (check_screen('generic-desktop', 30) || ++$timeout > 10) {
+        # Now it will install required language packages and exit
+        # Put in the loop, because sometimes button is not pressed
+        wait_screen_change { send_key 'alt-o'; };
+        sleep 60;
+        send_key 'ctrl';
+    }
+
+    # Logout and login to ensure the configuration for Chinese language take effect.
+    handle_logout;
+    handle_login;
```

Comment:
> handle_logout;
> handle_login;

The codes can be replaced by `handle_relogin;`

Reply:
Ok, thanks.Verified with https://openqa.nue.suse.com/tests/7714601#.